### PR TITLE
Add deposit and withdraw

### PR DESCRIPTION
**Pull request by Sarahb-da**
adds ability for investor to make deposit and withdraw requests

### CODE REVIEW

File: ui/src/components/common/WalletTransaction.tsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    const registeredCustodians = useStreamQueryAsPublic(RegisteredCustodian).contracts
+        .map(makeContractInfo)
+
+    const allTokens = useStreamQueries(Token, () => [], [], (e) => {
+            console.log("Unexpected close from Token: ", e);
+        }).contracts.map(makeContractInfo)
+
+    const { step, placeholder } = preciseInputSteps(Number(token?.contractData.quantityPrecision));
+
+    const handleSetDepositQuntity = (event: React.SyntheticEvent, result: any) => {
```
Comment (alexgraham-da) [inline ui/src/components/common/WalletTransaction.tsx:63]: "Quntity" 👍

**Review by alexgraham-da — APPROVED** [review]
Looks good!

File: ui/src/components/common/Holdings.tsx [hunk 2]
```diff
@@ -53,23 +53,24 @@ const Holdings: React.FC<Props> = ({ deposits, providers, role }) => {
...
-                </p>
-            </div>
-        )
-    }
+export function getProviderLabel(assetLabel: string, providers: DepositProvider[]) {
+    const providerInfo = providers.find(p => p.party === assetLabel)
+    return (
+        <div className='provider-info'>
+            <Header as='h5'>
+                {providerInfo?.label.substring(providerInfo.label.lastIndexOf('|')+1)}
```
Comment (alexmatson-da) [inline ui/src/components/common/Holdings.tsx:67]: 👍

**Review by alexmatson-da — APPROVED** [review]
Awesome!

**Pull request merged**